### PR TITLE
Add new landing page starter for next-js

### PR DESCRIPTION
**Pull request by teleaziz**

### CODE REVIEW

File: examples/next-js/headless-shopify/README.md [hunk 1]
```diff
@@ -0,0 +1 @@
+For the complete starter kit for Shopify + Next.js + Builder check out the starter [here](https://github.com/BuilderIO/nextjs-shopify)
```
Comment (steve8708) [inline examples/next-js/headless-shopify/README.md:1]: great idea

**Review by steve8708 — APPROVED** [review]
this is amazing!

**Review by mrkoreye — APPROVED** [review]
🎉

File: examples/next-js/landing-page/pages/[[...path]].tsx [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+  params,
+}: GetStaticPropsContext<{ path: string[] }>) {
+  const page = await resolveBuilderContent('page', {
+    urlPath: '/' + (params?.path?.join('/') || ''),
+  })
+
+  return {
+    props: {
+      page,
+    },
```
Comment (steve8708) [inline examples/next-js/landing-page/pages/[[...path]].tsx:25]:
```suggestion
    },
    notFound: !page,
```

it's not very well documented but it's supported for "fallback: true" a a top level option

**Comment by steve8708** [comment]
just realized you nested everything in `next-js`

imo I think it's best to keep things flat like I see in a lot of other repos, it's nice to see at a glance that there is all kinds of next-js examples

examples
  next-js-simple
  next-js-shopify

etc

**Comment by steve8708** [comment]
fuck sorry didn't mean to close the PR I hate how big that button is

**Comment by steve8708** [comment]
also I would just call the `landing-page` one `simple` - all starters support landing pages, we want people to choose this one for a simple / minimal example

File: examples/next-js/landing-page/README.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
+Next.js + Builder.io Landing page starter kit.
```
Comment (steve8708) [inline examples/next-js/landing-page/README.md:1]:
```suggestion
# Next.js + Builder.io Minimal Starter
```

File: examples/next-js/landing-page/README.md [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+npm install
+```
+
+### Run the dev server
+```
+npm run dev
+```
+A development server will start on `http://localhost:3000`
+
+Go to your [new space settings](https://builder.io/account/space) and change the site url to your localhost `http://localhost:3000` for site editing.
```
Comment (steve8708) [inline examples/next-js/landing-page/README.md:29]:
```suggestion
Go to your [new space settings](https://builder.io/account/space) and change the site url to your localhost `http://localhost:3000` for site editing. [Learn more about setting up preview URLs](https://www.builder.io/c/docs/guides/preview-url)
```

**Pull request merged**